src/components/inputs/Dates/DateRange/DateRange.props.story.tsx

```diff
@@ -52,6 +52,14 @@ export function MaxDate() {
   );
 }
 
+export function Locale() {
+  return (
+    <Layout.StoryVertical center>
+      <DateRangeButton locale={'ko-KR'} />
```

Comment:
Please add the ability to toggle locales in this story.

Reply:
do you have an example of what you mean? there's a select for supported locales but not sure if that's sufficient
<img width="1064" alt="Screen Shot 2022-04-29 at 11 51 27 AM" src="https://user-images.githubusercontent.com/52298236/165979929-130a7b47-c939-4430-92bc-37908339f205.png">

Reply:
So sorry! I was looking at the `Calendar` story which doesn't have controls. Please disregard 😅